Model.js

```diff
@@ -0,0 +1,47 @@
+const knex = require('knex');
+
+const knexConfig = require('./knexfile.js');
+const db = knex(knexConfig.development);
+
+module.exports = {
+  find,
+  findById,
+  add,
+  update,
+  remove,
+};
+
+function find() {
+  return db('dishes');
+}
```

Comment:
Follow the README when naming these functions. For example, this should be called `getDishes()`.